CDC-Data-Reconciliation-Backend/compare.py

```diff
@@ -66,7 +86,7 @@ def get_cdc_dict(cdc_file):
 
     return cdc_dict
 
-
+# place the stats stuff here
 def comp(state_dict, cdc_dict):
     for state_case_id in state_dict:
         state_row = state_dict[state_case_id]
```

Comment:
After line 92 we should check whether the 'CaseClassStatus' is 'Confirmed' or 'Suspect', if it isn't we should skip this entry for comparison / stats. We should talk to Marion on whether 'Suspect' cases are to be reported to the CDC as well.

I believe you should create a new entry to stats if the event code is not yet in the stats dictionary. If it already is in the dictionary, add to 'totalCases'.